yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+  - name: wind_speed
+    "@id": "#Exposure.wind_speed"
+    datatype: float
+    description: Outside wind speed.
+  - name: wind_dir
+    "@id": "#Exposure.wind_dir"
+    datatype: float
+    description: Wind direction.
+  - name: dimm_seeing
+    "@id": "#Exposure.dimm_seeing"
+    datatype: float
+    description: Seeing as measured by external DIMM.
+  - name: shut_lower
+    "@id": "#Exposure.shut_lower"
+    datatype: float
+    description: Dome dropout door opening percentage
+  - name: shut_upper
+    "@id": "#Exposure.shut_upper"
+    datatype: float
+    description: Dome main door opening percentage
+  - name: focus_z
+    "@id": "#Exposure.focus_z"
+    datatype: float
+    description: Focus Z position
+  - name: dome_azimuth
+    "@id": "#Exposure.dome_azimuth"
+    datatype: float
+    description: Dome azimuth
+  - name: simulated
+    "@id": "#Exposure.simulated"
+    datatype: boolean
+    description: Were any control system components simulated?
+  - name: exposureInOODS
+    "@id": "#Exposure.exposureInOODS"
+    datatype: boolean
+    description: Are all detectors ingested into the OODS?
+  - name: exposureInUSDF
+    "@id": "#Exposure.exposureInUSDF"
+    datatype: boolean
+    description: Are all detectors ingested into the USDF?
+- name: CcdExposure
+  "@id": "#CcdExposure"
+  columns:
+  - name: ccdExposureId
+    "@id": "#CcdExposure.ccdExposureId"
+    datatype: long
+    nullable: false
+    description: Primary key (unique identifier).
+  - name: exposure_id
+    "@id": "#CcdExposure.exposure_id"
+    datatype: long
+    nullable: false
+    description: Reference to the corresponding entry in the Exposure table.
+  - name: detector
+    "@id": "#CcdExposure.detector"
+    datatype: long
+    nullable: false
+    description: Detector ID. A detector associated with a particular instrument (not
+      an observation of that detector)
+  - name: xSize
+    "@id": "#CcdExposure.xSize"
+    datatype: long
+    description: Number of columns in the image.
+  - name: ySize
+    "@id": "#CcdExposure.ySize"
+    datatype: long
+    description: Number of rows in the image.
+  - name: llcra
+    "@id": "#CcdExposure.llcra"
+    datatype: double
+    description: RA of lower left corner.
+  - name: llcdec
+    "@id": "#CcdExposure.llcdec"
+    datatype: double
+    description: Declination of lower left corner.
+  - name: ulcra
+    "@id": "#CcdExposure.ulcra"
+    datatype: double
+    description: RA of upper left corner.
+  - name: ulcdec
+    "@id": "#CcdExposure.ulcdec"
+    datatype: double
+    description: Declination of upper left corner.
+  - name: urcra
+    "@id": "#CcdExposure.urcra"
+    datatype: double
+    description: RA of upper right corner.
+  - name: urcdec
+    "@id": "#CcdExposure.urcdec"
+    datatype: double
+    description: Declination of upper right corner.
+  - name: lrcra
+    "@id": "#CcdExposure.lrcra"
+    datatype: double
+    description: RA of lower right corner.
+  - name: lrcdec
```

Comment:
Also turning all the corners into an `s_region` string makes the table immediately more usable in client software.